testing/trino-product-tests/src/main/java/io/trino/tests/iceberg/TestSparkCompatibility.java

```diff
@@ -181,7 +181,7 @@ public void testSparkReadsPrestoPartitionedTable()
     @Test(groups = {ICEBERG, PROFILE_SPECIFIC_TESTS})
     public void testPrestoReadsSparkPartitionedTable()
     {
-        String baseTableName = "test_spark_reads_presto_partitioned_table";
+        String baseTableName = "test_presto_reads_spark_partitioned_table";
```

Comment:
Actually I thought concurrency could be the issue but it seems not. Let me instead change this commit message from `Fix test table name to avoid concurrency issue` to just `Fix typo`. We can add `randomTableSuffix()` later if concurrency issues come up.